.github/workflows/docker.yml

```diff
@@ -1,40 +1,45 @@
 name: ci
 
 on:
+  pull_request:
```

Comment:
I'm not sure if docker image should published by `pr`, generally is when new release is created or push to main branch
Example: [Starport](https://github.com/tendermint/starport/blob/develop/.github/workflows/release-docker.yml)

Reply:
My reasoning was that quite often when I'm reviewing a PR I want to really quickly run it and see if it works. I guess it's only slightly quicker than checking out the branch though, so happy to remove - your call dude